CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
 x.y.z Release notes (yyyy-MM-dd)
```

Comment:
This is not the correct way to fill out the CHANGELOG. You're missing the version, the compatibility is wrong (which is particularly important in this PR, since we're breaking compat), and internal is wrong. Please see how other CHANGELOG entries are filled in.